etw/java/build.gradle

```diff
@@ -0,0 +1,122 @@
+/*
+ * ApplicationInsights-Java
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ * software and associated documentation files (the ""Software""), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+ * persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+ * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+plugins {
+    id 'java-library'
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply from: "$buildScriptsDir/common-java.gradle"
+apply from: "$buildScriptsDir/provided-configuration.gradle"
+
+configurations {
+    jni64 {
+        transitive = false
+    }
+    jni32 {
+        transitive = false
+    }
+    provided {
+        transitive = false
+    }
+    testRuntime.extendsFrom provided
+}
+
+def jniSrcDir = 'src/main/jni'

```

Comment:
I don't see this directory (or anything else in this module)?

Reply:
`etw/java/src/main/jni`. which module?

Reply:
Sorry, github UI failing me (and I'm failing it).